### PR TITLE
fix: use storage CRS code from list of supported CRS

### PR DESCRIPTION
**Pull request by stempler**
...instead of the CRS name.
If no match can be found for the storage CRS codes in the list of supported CRS, the first of the storage CRS codes is used.

This is done to if possible fulfill the requirement in the specification of OGC API Features Part 2, that the storage CRS shall be one of the identifiers of the supported CRS.

### CODE REVIEW

**Comment by stempler** [comment]
Reverted the workaround for the build not working as well.

File: deegree-ogcapi-config-htmlview/pom.xml [hunk 1]
```diff
@@ -22,11 +22,11 @@
 
   <dependencies>
     <dependency>
-      <groupId>${deegree-group}</groupId>
+      <groupId>org.deegree</groupId>
```
Comment (kapil-agnihotri) [inline deegree-ogcapi-config-htmlview/pom.xml:25]: For my curiosity, why the variable `${deegree-group}` is replaced with a constant `org.deegree`?

File: deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/workspace/DeegreeDataAccess.java [hunk 2]
```diff
@@ -231,11 +231,45 @@ private Collection createCollection( OafDatasetConfiguration oafConfiguration, S
...
+     * @param suppportedCodes the supported/preferred codes
+     * @return the selected code
+     */
+    public static String selectCrsCode(List<String> crsCodes, List<String> suppportedCodes) {
+    	if (crsCodes == null || crsCodes.isEmpty()) {
+    		return null;
+    	}
+    	
+		for (String candidate : crsCodes) {
+			if (suppportedCodes.contains(candidate)) {
```
Comment (kapil-agnihotri) [inline deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/workspace/DeegreeDataAccess.java:265]: Any chance of `suppportedCodes` being `null`?

**Review by kapil-agnihotri — APPROVED** [review]

File: deegree-ogcapi-config-htmlview/pom.xml (hunk 1 above)
Comment (stempler) [reply deegree-ogcapi-config-htmlview/pom.xml:25]: That commit is just reverting this one: 02f41d5167bc88b6d9de03332d03127920ee53d2

File: deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/workspace/DeegreeDataAccess.java (hunk 2 above)
Comment (stempler) [reply deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/workspace/DeegreeDataAccess.java:265]: The origin of the list can be tracked back to `OafResource#parseQueryCrs` which never returns `null`.

**Pull request merged**